### PR TITLE
Update did veres one

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "did-cli",
   "version": "0.4.2-0",
   "description": "A program for managing Decentralized Identifiers (DIDs).",
+  "license": "SEE LICENSE IN LICENSE.md",
```
Comment (davidlehn) [inline package.json]: It's just 'LICENSE' in this project, not a md file.

Comment (davidlehn) [reply package.json]: Or in this case, perhaps it should just be the proper BSD tag rather than this note.

Comment (dmitrizagidulin) [inline package.json]:
```suggestion
  "license": "BSD",  
```

Comment (dmitrizagidulin) [inline package.json]:
```suggestion
  "license": "BSD-3-Clause",  
```

Comment (mattcollier) [reply package.json]: Fixed.

File: package.json [hunk 2]
```diff
@@ -25,7 +26,7 @@
   "dependencies": {
     "chalk": "^2.4.2",
     "crypto-ld": "^3.7.0",
-    "did-veres-one": "^8.0.0",
+    "did-veres-one": "veres-one/did-veres-one#update-context",
```
Comment (dmitrizagidulin) [inline package.json:29]:
```suggestion
    "did-veres-one": "^9.0.0",
```

Comment (dmitrizagidulin) [reply package.json:29]: did-veres-one@9 should now have the updated context, and Tobias's assertMethod feature.

**Comment by dmitrizagidulin** [comment]
Merging into `update-to-latest`. (I'll bump the dependency to `v9.0.0` in that branch.)

**Review by dmitrizagidulin — APPROVED** [review]
👍

**Pull request merged**